login.html

```diff
@@ -1,107 +1,133 @@
 <!DOCTYPE html>
 <html>
-	<head>
```

Comment:
Main edits for this version were indenting. Version 2 of the login page was changed mostly in CSS